resources/template.html

```diff
@@ -140,22 +139,31 @@ <h2 class="mb-0">
 				</tr>
 			</table>
 			{{- else}}
+                        {{- $h := .Histogram }}
 			{{- with .Histogram}}
 			<table class="table table-striped table-bordered">
-				{{- range .Bucket}}
+                                {{- if .Schema }}
 				<tr>
-					<th scope="row">Sample values &le; {{value .GetUpperBound}}</th>
-					<td>{{.GetCumulativeCount}}</td>
-				</tr>
-				{{- end}}
-				<tr>
-					<th scope="row">Total sample Count</th>
-					<td>{{.GetSampleCount}}</td>
-				</tr>
-				<tr>
-					<th scope="row">Sample Sum</th>
-					<td>{{value .GetSampleSum}}</td>
+                                    <th scope="row">Native Histogram representation</th>
+                                    <td>{{formatHistogram $h}}</td>
 				</tr>
+                                {{- end}}
+                                {{- if gt (len .Bucket) 0}}
```

Comment:
Or if there are neither native nor classic buckets (to handle the case of a bucket-less histogram).